cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -228,6 +228,11 @@ func ReadinessProbe(host, path string, port int, scheme v1.URIScheme) *v1.Probe
 	return createHTTPProbe(host, path, port, scheme, 0, 15, 3, 1)
 }
 
+// StartupProbe creates a Probe object with a HTTPGet handler
+func StartupProbe(host, path string, port int, scheme v1.URIScheme) *v1.Probe {
+	return createHTTPProbe(host, path, port, scheme, 15, 15, 30, 10)
```

Comment:
Could you add a short comment, that we will wait 5 minutes until the API server comes up?